data-prepper-plugins/aws-plugin/src/main/java/org/opensearch/dataprepper/plugins/aws/AwsSecretPlugin.java

```diff
@@ -33,6 +35,7 @@ public AwsSecretPlugin(final AwsSecretPluginConfig awsSecretPluginConfig) {
             this.pluginConfigPublisher = new AwsSecretsPluginConfigPublisher();
             pluginConfigValueTranslator = new AwsSecretsPluginConfigValueTranslator(secretsSupplier);
             scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
+            pluginMetrics = PluginMetrics.fromNames("secrets", "aws");
```

Comment:
I tend to think that this should be `fromNames("aws", "secrets")`

I'm not sure that we will even have other secrets, so this seems to create an unnecessary extra layer.

Reply:
This is to be consistent with https://github.com/opensearch-project/data-prepper/blob/e6df3eb2cd46ebd13dd1c7b808d288c2f3d6ee51/data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/certificate/CertificateProviderFactory.java#L68

Reply:
The documentation indicates that we go from a componentId to a componentScope.

https://github.com/opensearch-project/data-prepper/blob/1af1ce9c6684994aaac059a4f79b8a780f8a09a8/data-prepper-api/src/main/java/org/opensearch/dataprepper/metrics/PluginMetrics.java#L36-L37

I think the component in this case is the aws-plugin. So I would be in favor of using `aws` as the componentId. 

@asifsmohammed , Any thoughts from you end?

Reply:
I think what George has in the PR is right order. We do `<componentScope> + "." + <componentId>`